### PR TITLE
feat: add paytm credentials and setup in admin/settings

### LINKED ISSUE #3320: Addition of PayTM keys in payment gateways
Parent Issue : #3319 
Implement PayTM enabling once the server issue has been taken care of

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #3320 

#### Changes proposed in this pull request:

- Add PayTM credentials in settings model.
- Add validations for new fields in the project.
- Add fields following other payment gateways

Screenshot from 2019-08-10 13-28-37

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/admin/settings/payment-gateway-form.js [hunk 1]
```diff
@@ -213,6 +257,14 @@ export default Component.extend(FormMixin, {
             'omiseLiveSecret' : null
           });
         }
+        if (!this.isCheckedPaytm) {
```
Comment (kushthedude) [inline app/components/forms/admin/settings/payment-gateway-form.js]: Why is this needed, this will set all properties to null once I disable payTM for a short time and also `is_payTM_activated` boolean would be enough for this.

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 2]
```diff
@@ -237,6 +236,73 @@
       </div>
     {{/if}}
   {{/if}}
+  <div class="inline field">
+    <label>
+      {{t 'Enable PayTM'}}
+    </label>
+    {{ui-checkbox class='toggle' checked=isCheckedPaytm onChange=(action (mut isCheckedPaytm))}}
```
Comment (kushthedude) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: `isPaytmActivated`?

File: app/components/forms/admin/settings/payment-gateway-form.js [hunk 3]
```diff
@@ -181,6 +221,10 @@ export default Component.extend(FormMixin, {
     return this.get('settings.alipaySecretKey') && this.get('settings.alipayPublishableKey');
   }),
 
+  isCheckedPaytm: computed(function() {
```
Comment (kushthedude) [inline app/components/forms/admin/settings/payment-gateway-form.js]: No need for this

**Comment by mrsaicharan1** [comment]
@uds5501 So just for clarification, we're trying to enable scenarios where paytm can be disabled and keys are retained. Hope that helps! I've made the changes on the server too. So you can check it out.

**Comment by uds5501** [comment]
@kushthedude @mrsaicharan1 took care of these in the current iteration, please check

**Comment by uds5501** [comment]
The server PR : fossasia/open-event-server#6249 has been merged. 
@mrsaicharan1  please review and merge
